inconsistencies.md

```diff
@@ -0,0 +1,153 @@
+# Solving Cassandra consistency
+
+This plan makes the following assumption:
+
+  - We are not willing to rewrite mailbox/cassandra with event sourcing
```

Comment:
"because we think we can find solutions that are cheaper that, without being perfect, would allow reduce inconsistency"

That's it?

Reply:
That's it.

I would complete: 
 - because I think event sourcing with a cassandra tooling don't allow to realistically implement mailbox/cassandra.

Reply:
> because I think event sourcing with a cassandra tooling don't allow to realistically implement mailbox/cassandra.

It would be an valid argument if we had a realistic plan B.

BTW, It's not because you don't know yet how to implement it with Event Sourcing that it's not realistic. Given the complexity of problems we have to face without Event Sourcing, I'm confident to say that it would probably be easier (but less incremental) to jump in the ES boat.

Let's see if that plan works

Reply:
> It would be an valid argument if we had a realistic plan B.

I'm trying to propose a realistic (incremental) plan B.

Reply:
Of course.

I just want to document why we didn't consider ES so that we can challenge those reasons later.